azure-quantum/azure/quantum/target/honeywell.py

```diff
@@ -0,0 +1,56 @@
+##
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+##
+import io
+
+from azure.quantum.target.target import Target
+from azure.quantum.job.job import Job
+from azure.quantum.workspace import Workspace
+
+
+class Honeywell(Target):
+    """Honeywell target."""
+
+    def __init__(
+        self,
+        workspace: Workspace,
+        target: str = "honeywell.hqs-lt-s1-apival",
```

Comment:
I set the `ionq.simulator` target as the default for IonQ as well so I thought it would be good to be consistent. Do you think this should simply have no default value?
Nonetheless I think we still need a `Workspace.list_providers()` method that lists all the accessible providers and targets.